extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1498,13 +1502,18 @@ export class AmpA4A extends AMP.BaseElement {
    * TODO(keithwrightbos@): report error cases
    */
   getAmpAdMetadata_(creative) {
```

Comment:
Yes, I'm adding a variant. The idea is to make it the only variant (it's the simplest, most orderly variant), but to do this we'll need to change the server side, once this change is released, and then come back.
The reason why sorting the attribute names is useful is that we do it for everything else on the server side now. So this thing sticks out like a sore thumb, and could break by accident.